### PR TITLE
#175: The copy address button should be added back

### LINKED ISSUE #175: The copy address button should be added back
As in https://github.com/skycoin/skycoin/issues/1575, the copy address button should be added back to the wallet screen. However, the copy address button should not be removed from the menu.

### PR DESCRIPTION
**Pull request by alexandermushinskiy**
Fixes #175 

Changes:
- added back the copy wallet address button

### CODE REVIEW

File: src/app/components/pages/wallets/address-detail/wallet-detail.component.scss [hunk 1]
```diff
@@ -123,3 +127,66 @@
...
   }
 }
+
+@keyframes floatup {
+  20% {
+    opacity: .999
+  }
+
+  100% {
+    -webkit-transform: translate3d(-50%, -17px, 0);
```
Comment (Senyoret1) [inline src/app/components/pages/wallets/address-detail/wallet-detail.component.scss]: This line is not necessary, the Sass compiler is responsible for placing the prefix.

File: src/app/components/pages/wallets/address-detail/wallet-detail.component.scss [hunk 2]
```diff
@@ -123,3 +127,66 @@
...
+  }
+}
+
+@-webkit-keyframes floatup {
+  20% {
+    opacity: .999
+  }
+
+  100% {
+    -webkit-transform: translate3d(-50%, -17px, 0);
```
Comment (Senyoret1) [inline src/app/components/pages/wallets/address-detail/wallet-detail.component.scss]: This line is not necessary, the Sass compiler is responsible for placing the prefix.

File: src/app/components/pages/wallets/address-detail/wallet-detail.component.scss [hunk 3]
```diff
@@ -123,3 +127,66 @@
...
+  &::after {
+    content: attr(data-label);
+    color: #0072ff;
+    display: inline-block;
+    position: absolute;
+    top: -2px;
+    left: 50%;
+    opacity: .001;
+    text-align: center;
+    -webkit-transform: translate3d(-50%,0,0);
```
Comment (Senyoret1) [inline src/app/components/pages/wallets/address-detail/wallet-detail.component.scss]: This line is not necessary, the Sass compiler is responsible for placing the prefix.

File: src/app/components/pages/wallets/address-detail/wallet-detail.component.scss [hunk 4]
```diff
@@ -123,3 +127,66 @@
...
+    transform: translate3d(-50%,0,0);
+    -webkit-backface-visibility: hidden;
```
Comment (Senyoret1) [inline src/app/components/pages/wallets/address-detail/wallet-detail.component.scss]: Change to `backface-visibility: hidden;`, the Sass compiler is responsible for placing the prefix.

File: src/app/components/pages/wallets/address-detail/wallet-detail.component.scss (hunk 1 above)
Comment (alexandermushinskiy) [reply src/app/components/pages/wallets/address-detail/wallet-detail.component.scss]: Fixed.

File: src/app/components/pages/wallets/address-detail/wallet-detail.component.scss (hunk 3 above)
Comment (alexandermushinskiy) [reply src/app/components/pages/wallets/address-detail/wallet-detail.component.scss]: Fixed.

File: src/app/components/pages/wallets/address-detail/wallet-detail.component.scss (hunk 4 above)
Comment (alexandermushinskiy) [reply src/app/components/pages/wallets/address-detail/wallet-detail.component.scss]: Fixed.

**Comment by alexandermushinskiy** [comment]
Review comments are fixed.

**Pull request merged**